### ISSUE TITLE
Multiple Sauce Connect executables started when running tests in parallel

### ISSUE THREAD

**Issue by bootstraponline** [saucelabs/sauce_ruby#318]
`ps -ef | awk '/bin\/sc/ {print $0}'` shows two SC executables when only one is expected.

**Comment by DylanLacey** [comment]
Hmm.  Perhaps the way we're using Parallel Tests' first process only to start SC isn't correct anymore.  Were you experiencing tunnel stability problems?

**Comment by bootstraponline** [comment]
Yes, the SC feature in the Ruby gem is broken. Among the issues is it'll infintie loop and spawn more than one SC executable.

**Comment by bootstraponline** [comment]
Also it crashes. https://github.com/saucelabs/sauce_ruby/issues/317
